common/setups/rasr/hybrid_system.py

```diff
@@ -368,6 +368,7 @@ def nn_recognition(
         use_epoch_for_compile=False,
         forward_output_layer="output",
         native_ops: Optional[List[str]] = None,
+        acoustic_mixture_path: Optional[tk.Path] = None,
```

Comment:
This is not used afterwards right? This was removed. You probably had to add it because there is some call which includes this (c.f. other comment further below)
Lets better fix the wrong call.